crates/testenv/src/lib.rs

```diff
@@ -20,12 +20,12 @@ pub struct TestEnv {
 impl TestEnv {
     /// Construct a new [`TestEnv`] instance with default configurations.
     pub fn new() -> anyhow::Result<Self> {
-        let bitcoind = match std::env::var_os("BITCOIND_EXE") {
+        let bitcoind = match std::env::var_os("BITCOIND_EXEC") {
             Some(bitcoind_path) => electrsd::bitcoind::BitcoinD::new(bitcoind_path),
             None => {
                 let bitcoind_exe = electrsd::bitcoind::downloaded_exe_path()
                     .expect(
-                "you need to provide an env var BITCOIND_EXE or specify a bitcoind version feature",
+                "you need to provide an env var BITCOIND_EXEC or specify a bitcoind version feature",
```

Comment:
These were driving me crazy. It was not trivial to find.
These were wrong according to the `bitcoind`/`electrsd` docs.
Glad that I was able to fix it.

Cc @LagginTimes.